dvc/repo/graph.py

```diff
@@ -158,10 +158,9 @@ def build_outs_graph(graph, outs_trie):
 
     outs_graph.add_nodes_from(outs_trie.values())
     for stage in graph.nodes():
+        if stage.is_repo_import:
+            continue
```

Comment:
I think this check could actually be `if stage.is_import` here since outs can't overlap with external dep paths now?

(But I left the check as repo imports only for now to match the original behavior, and since import-url deps won't cause the erepo git cloning issue)